packages/v4/patternfly-docs/content/accessibility/menu-toggle/menu-toggle.md

```diff
@@ -10,7 +10,7 @@ import { Link } from '@patternfly/documentation-framework/components/link/link';
 ## Accessibility
 
 To implement an accessible PatternFly **menu toggle** component:
-- Due to the composable nature of the menu and menu toggle components, keyboard interaction will need to be added manually between the menu toggle and the menu. See the [composable menu demos](/demos/composable-menu) to see how this is done.
+- Due to the composable nature of the menu and menu toggle components, keyboard interaction will need to be added manually between the menu toggle and the menu. See the [composable menu demos](/components/menus/custom-menus) to see how this is done.
```

Comment:
Let's also update the link text here to read `custom menu demos` rather than `composable menu demos`

Reply:
Really nitty nit - that page is titles "Custom menus" plural so we should make that match. :-)

Reply:
@srambach fixed!